### DISCUSSION TITLE
Health Commands

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#3310] [Unsupported Versions] [answered]
What are commands to confirm the health of my security onion installation of all nodes?

**Comment by facyber** [comment, marked as answer]
Use `so-status` on each node.

**Reply by TOoSmOotH** [reply]
Or one could do:

`sudo salt \* so.status
`

**Comment by stondino00** [comment]
Thanks.  Is there a more detailed health report command to run on each node?

**Reply by facyber** [reply]
You can get some data with `docker stats`. Google more about docker container health checks.

**Reply by TOoSmOotH** [reply]
so-status will tell you if everything is running. Use grafana if you want detailed information about performance.
